src/System.Private.ServiceModel/tests/Scenarios/Contract/Fault/FaultExceptionTests.4.0.0.cs

```diff
@@ -299,4 +301,74 @@ public static void FaultException_FaultContractAndKnownType_Echo()
             ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
         }
     }
+
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+#endif
+    [WcfFact]
+    [OuterLoop]
+    public static void TestAccessFaultContractInfo()
```

Comment:
I think we need to move this test out of the 4.0.0.cs file and into FaultExceptionTests.cs (maybe need to create a new file for it).  This API didn't exist in 4.0.0, so this test falls into the category of "pre-release".